docs/provisioning_aws_cluster.md

```diff
@@ -0,0 +1,88 @@
+# Provisioning an AWS Cluster
+
+In this doc we describe how to provision a new AWS OCP cluster via ACM
+
+The reference images describe the settings when using the AWS OCTO provider.
+
+### Pre-requisite
+Must be an Operate-First admin, with cluster-admin access to ACM
+
+
+### Steps:
+
+Navigate to https://multicloud-console.apps.moc-infra.massopen.cloud/multicloud/clusters
+
+Click "Create Cluster"
+
+![][1]
+
+#### Infrastructure Provider
+Select "Red Hat OpenShift Container Platform" as the Kubernetes distribution
+Select "Amazon Web Services" as the infrastructure provider
+Select the appropriate "Infrastructure Provider Credentials" from the drop down (`aws-octo` for the OCTO aws account)
+Click Next
+
+![][2]
+
+
+#### Cluster Details
+Choose an appropriate `cluster name`, keep in mind this name will appear on all the routes.
+Leave `cluster set` and `additional labels` blank.
+Choose the latest release image.
+Base DNS domain will be auto filled in based on the configured provider you selected (`aws.operate-first.cloud` for OCTO aws account).
+
+![][3]
+
+#### Master Node
+Select the appropriate aws region to deploy the master nodes to (`us-west-2` for OCTO aws account).
```

Comment:
The "proper" regions are `eu-east-2` aka **"Europe (London)"** or `us-east-2` aka **"US East (Ohio)"**